<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Geotag</class>
 <widget class="QWidget" name="Geotag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>GPSLongitude:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lngLineEdit">
     <property name="inputMask">
      <string>00.000000; </string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QGridLayout" name="mapGrid"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="latLineEdit">
     <property name="inputMask">
      <string>00.000000; </string>
     </property>
     <property name="maxLength">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>GPSLatitude:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="gotoButton">
     <property name="text">
      <string>goto</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="zoomInButton">
       <property name="text">
        <string>zoom in</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomOutButton">
       <property name="text">
        <string>zoom out</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="geotagCheckBox">
     <property name="text">
      <string>Change Geotag in Exif data</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
